### PR TITLE
Fix checkstyle violations in 5 `transcription-service*` modules

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
Work on #2238

- `transcription-service-amberscript`
- `transcription-service-google-speech-impl`
- `transcription-service-ibm-watson-impl`
- `transcription-service-persistence`
- `transcription-service-workflowoperation`

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/#development-process/#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: modules/transcription-service-amberscript/src/main/java/org/opencastproject/transcription/amberscript/AmberscriptTranscriptionService.java [hunk 1]
```diff
@@ -567,9 +570,9 @@ public MediaPackageElement getGeneratedTranscription(String mpId, String jobId)
         workspace.get(uri);
         logger.info("Found captions at URI: {}", uri);
       } catch (Exception e) {
-          logger.info("Results not saved: getting from service for jobId {}", jobId);
+        logger.info("Results not saved: getting from service for jobId {}", jobId);
           // Not saved yet so call the transcription service to get the results
```
Comment (lkiesow) [inline modules/transcription-service-amberscript/src/main/java/org/opencastproject/transcription/amberscript/AmberscriptTranscriptionService.java]:
```suggestion
        // Not saved yet so call the transcription service to get the results
```

File: modules/transcription-service-google-speech-impl/src/main/java/org/opencastproject/transcription/googlespeech/endpoint/GoogleSpeechTranscriptionRestService.java [hunk 2]
```diff
@@ -112,10 +131,23 @@ public Response getTranscriptionResult(@QueryParam("jobId") String jobId,
...
-    @RestResponse(responseCode = HttpServletResponse.SC_OK, description = "If no errors"),
-    @RestResponse(responseCode = HttpServletResponse.SC_INTERNAL_SERVER_ERROR, description = "An error occurred")})
+  @RestQuery(
+      name = "status",
+      description = "Get mediapackage transcription status",
+      returnDescription = "Returns transcription status",
+      restParameters = {
+          @RestParameter(
+              name = "mediaPackageID",
+              description = "the mediapackage identifier",
```
Comment (lkiesow) [inline modules/transcription-service-google-speech-impl/src/main/java/org/opencastproject/transcription/googlespeech/endpoint/GoogleSpeechTranscriptionRestService.java]:
```suggestion
              description = "The mediapackage identifier",
```

File: modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java [hunk 3]
```diff
@@ -471,6 +478,7 @@ protected String process(Job job) throws Exception {
     return result;
   }
 
+  // CHECKSTYLE:OFF
```
Comment (lkiesow) [inline modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java:482]: Can we add why we turn this off?
Something like (not tested):

```suggestion
  // CHECKSTYLE:OFF  Example URL is too long but needs to stay like this
```

File: modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java [hunk 4]
```diff
@@ -526,13 +537,15 @@ void registerCallback() throws TranscriptionServiceException {
...
-        if (response != null)
+        if (response != null) {
           response.close();
+        }
       } catch (IOException e) {
       }
     }
   }
 
+  // CHECKSTYLE:OFF
```
Comment (lkiesow) [inline modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java:550]: same here

File: modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java [hunk 5]
```diff
@@ -965,8 +986,9 @@ public void run() {
             String jobId = j.getTranscriptionJobId();
             if (wfId == null) {
               logger.warn(
-                      "Retry transcription workflow could NOT be scheduled for mp {}, watson job {}. Will try again next time.",
-                      mpId, jobId);
+                  "Retry transcription workflow could NOT be scheduled for mp {}, watson job {}. "
```
Comment (lkiesow) [inline modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java:993]:
```suggestion
                  "Retry transcription workflow could NOT be scheduled for media package {}, watson job {}. "
```

File: modules/transcription-service-workflowoperation/src/main/java/org/opencastproject/transcription/workflowoperation/AmberscriptAttachTranscriptionOperationHandler.java [hunk 6]
```diff
@@ -129,22 +128,25 @@ public WorkflowOperationResult start(final WorkflowInstance workflowInstance, Jo
...
+      MediaPackageElement vttElement
+          = builder.newElement(Attachment.TYPE, new MediaPackageElementFlavor("captions", "vtt"));
+      vttElement.setIdentifier(UUID.randomUUID().toString());
+      vttElement.setMimeType(MimeType.mimeType("text", "vtt"));
+      URI vttURI = workspace.put(mediaPackage.getIdentifier().toString(),
+          vttElement.getIdentifier(), "captions.vtt", zis);
+      vttElement.setURI(vttURI);
+      mediaPackage.add(vttElement);
 
         // Set the target flavor if informed
```
Comment (lkiesow) [inline modules/transcription-service-workflowoperation/src/main/java/org/opencastproject/transcription/workflowoperation/AmberscriptAttachTranscriptionOperationHandler.java]:
```suggestion
      // Set the target flavor if informed
```

**Review by lkiesow — CHANGES_REQUESTED** [review]

File: modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java (hunk 3 above)
Comment (LukasKalbertodt) [reply modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java:482]: In the default configuration, there can't be anything after `CHECKSTYLE:OFF`. One could configure that, but for now I just added a comment above it. Should be good for now.

**Comment by LukasKalbertodt** [comment]
I think I fixed all review remarks.

**Pull request merged**